.github/workflows/close-stale-issues.yml

```
@@ -0,0 +1,37 @@
+name: stale
+
+on:
+  schedule:
+    - cron: '30 1 * * *'
```

Does not hurt to add a `workflow_dispatch` event if we want to trigger that workflow manually to make sure everything looks good:

```suggestion
  workflow_dispatch:
  schedule:
    - cron: '30 1 * * *'
```